config.yml

```diff
@@ -22,11 +22,11 @@ sleep-timeout: 3
 # script location to manipulate node power cycle
 node-scripts-location: 'scripts/vagrant'
 
-opensuse-image-url: https://download.opensuse.org/repositories/Cloud:/Images:/Leap_15.5/images/openSUSE-Leap-15.5.x86_64-NoCloud.qcow2
+opensuse-image-url: https://rsync.opensuse.org/repositories/Cloud%3A/Images%3A/Leap_15.6/images/openSUSE-Leap-15.6.x86_64-NoCloud.qcow2
```

Comment:
Any difference between _rsync.opensuse.org_ and _download.opensuse.org_?

Reply:
I looked up the most recent image locations and this was what showed up. It shouldn't really matter since they are both running as web servers serving the files. The only downside I see is that if the server changes we'll have to change the URL again. Also this URL doesn't use colons in the URL which should help a bit with compatibility.